src/index.html

```diff
@@ -7,12 +7,71 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__product-image"
+        src="./images/imac.jpeg"
+        alt="product-image"
+      />
+
+      <div class="card__product-details">
+        <p class="card__product-details-title">
```

Comment:
The element with class 'card__product-details-title' should be an 'h3' or another appropriate heading tag instead of 'p' to improve the semantic structure of the document.